specification/storsimple8000series/resource-manager/readme.md

```diff
@@ -56,3 +56,23 @@ csharp:
   output-folder: $(csharp-sdks-folder)/StorSimple8000Series/Management.StorSimple8000Series/Generated
   clear-output-folder: true
 ```
+
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: storsimple
+  clear-output-folder: true
+```
+
+### Tag: package-2017-06 and go
+
+These settings apply only when `--tag=package-2017-06 --go` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-06' && $(go)
+output-folder: $(go-sdk-folder)/services/storsimple8000series/mgmt/2017-06-01/storsimple

```

Comment:
>storsimple8000series/mgmt/2017-06-01/storsimple [](start = 41, length = 47)

Would it make more sense `foo/mgmt/date/foo` than `foobar/mgmt/date/foo`?

Reply:
Nope! It is better to keep the consistency of RP/version/RT than to make an exception for something that seems grammatically better.